dll/cpl/access/general.c

```diff
@@ -194,8 +195,10 @@ GeneralPageProc(HWND hwndDlg,
                            IDC_RESET_BOX,
                            pGlobalData->accessTimeout.dwFlags & ATF_TIMEOUTON ? BST_CHECKED : BST_UNCHECKED);
             FillResetComboBox(GetDlgItem(hwndDlg, IDC_RESET_COMBO));
-            SendDlgItemMessage(hwndDlg, IDC_RESET_COMBO, CB_SETCURSEL,
-                               (pGlobalData->accessTimeout.iTimeOutMSec / 300000) - 1, 0);
+            iCurSel = (pGlobalData->accessTimeout.iTimeOutMSec / 300000) - 1;
```

Comment:
I wonder why there was a "minus one" in the original code (it's obviously in cases where that timeout in milliseconds was strictly less than 300000 ms i.e. 300 seconds (!!), that the selection became == -1).

Reply:
See
https://github.com/reactos/reactos/blob/0ae6a509b0057d91c7ac67149025a1dd3ecc5d30/dll/cpl/access/general.c#L104
and
https://github.com/reactos/reactos/blob/0ae6a509b0057d91c7ac67149025a1dd3ecc5d30/dll/cpl/access/general.c#L236

While there, that cast should be `DWORD`, not `ULONG`, for consistency.

Reply:
> While there, that cast should be DWORD, not ULONG, for consistency.

correct.

Reply:
Fixed as `DWORD`.